crates/distribution-types/src/resolution.rs

```diff
@@ -10,19 +10,14 @@ use crate::{
 
 /// A set of packages pinned at specific versions.
 #[derive(Debug, Default, Clone)]
-pub struct Resolution(FxHashMap<PackageName, ResolvedDist>);
+pub struct Resolution(BTreeMap<PackageName, ResolvedDist>);
```

Comment:
This is typically not that large and only created ~once in the life of the program. Seems like a good tradeoff to make it deterministic.